shapeshift/web
# fix: asset search highlight and scrolling

## Description

Fixes a couple of small bugs I found in `web` during a refactor:

1. Set current account page / asset page asset `AssetRow` to active in the asset search component
2. Scroll current asset to the middle of `AssetList` search box

## Notice


## Pull Request Type


## Issue (if applicable)

N/A

## Risk

Minimal - affected code didn't work.

## Testing

1. Navigate to the Account page of an asset, e.g. FOX.
2. Click the asset search bar: FOX should scroll to the centre of the search results
3. Type FOX into the search bar: FOX should be highlighted (active)

## Screenshots (if applicable)

### Account Page

#### Native tokens

<img width="624" alt="image" src="https://user-images.githubusercontent.com/17035424/174397677-da3df26f-de85-4189-a303-ec9649221080.png">
<img width="625" alt="image" src="https://user-images.githubusercontent.com/17035424/174397633-a55399dd-88ef-43cc-9d35-d2fc61052cab.png">
<img width="626" alt="image" src="https://user-images.githubusercontent.com/17035424/174397761-a889242e-c4b4-437c-afb9-58959997849f.png">

#### ERC20s

<img width="621" alt="image" src="https://user-images.githubusercontent.com/17035424/174397811-a57e3ad3-6580-404e-9956-44c637fc97ea.png">

### Assets Page

#### Native tokens

<img width="622" alt="image" src="https://user-images.githubusercontent.com/17035424/174398072-ec67ea3f-396f-4125-a486-d0ca0500ca75.png">
<img width="624" alt="image" src="https://user-images.githubusercontent.com/17035424/174397910-efe8a288-b3f7-4d83-97ab-3c3aeb744957.png">
<img width="616" alt="image" src="https://user-images.githubusercontent.com/17035424/174397953-7ffd7ea8-c45a-46dc-8091-93304ab765f1.png">

#### ERC20s

<img width="622" alt="image" src="https://user-images.githubusercontent.com/17035424/174397990-1d2025d8-8949-4219-9ecd-d258cfa5738c.png">
